website/content/en/preview/upgrading/v1beta1-migration.md

```diff
@@ -185,7 +185,7 @@ The [`karpenter-convert`](https://github.com/aws/karpenter/tree/main/tools/karpe
     ```bash
     ROLE_NAME="${CLUSTER_NAME}-karpenter"
     POLICY_NAME="KarpenterControllerPolicy-${CLUSTER_NAME}"
-    POLICY_ARN=$(aws iam list-policies --query 'Policies[?PolicyName==`KarpenterControllerPolicy-scale-test`].Arn' --output text)
+    POLICY_ARN=$(aws iam list-policies --query "Policies[?PolicyName=='${POLICY_NAME}'].Arn" --output text)
```

Comment:
Good catch. I think this change may actually need to be made against the `main` brach. That is the branch where the versions of the docs are published from. You should be able to edit the docs on the `v0.32` path on the main branch to get the changes you want in upstream.

Reply:
As a release of version 0.32.x, a pull request was made at that point. Should I rewrite it and upload it to the main branch?